packages/sanity/playwright-ct/tests/formBuilder/PortableTextInput.spec.tsx

```diff
@@ -300,40 +290,44 @@ test.describe('Blocks', () => {
 
 test.describe('Menu bar', () => {
   test('Should display all default styles', async ({mount, page}, testInfo) => {
-    const {focusPTE} = testHelpers({page, testInfo})
+    const {getFocusedPortableTextInput} = testHelpers({page, testInfo})
     await mount(<PortableTextInputStory />)
-    const $pteField = await focusPTE('field-bodyStyles')
+    const $portableTextInput = await getFocusedPortableTextInput('field-bodyStyles')
 
     // Assertion: All icons in the menu bar should be visible
     const ICONS = ['bold', 'code', 'italic', 'link', 'olist', 'ulist', 'underline']
     for (const icon of ICONS) {
       await expect(
-        $pteField.getByRole('button').locator(`[data-sanity-icon="${icon}"]`),
+        $portableTextInput.getByRole('button').locator(`[data-sanity-icon="${icon}"]`),
       ).toBeVisible()
     }
 
     // Assertion: ???
-    await expect($pteField.locator('button#block-style-select')).not.toBeVisible()
+    await expect($portableTextInput.locator('button#block-style-select')).not.toBeVisible()
```

Comment:
**suggestion:** consider breaking this out into a separate test? E.g.

```
  test('Should not display block style button when no block styles are present', async ({
    mount,
    page,
  }, testInfo) => {
    const {getFocusedPortableTextInput} = testHelpers({page, testInfo})
    await mount(<PortableTextInputStory />)
    const $portableTextInput = await getFocusedPortableTextInput('field-bodyStyles')

    // Assertion: Block style button should be hidden
    await expect($portableTextInput.locator('button#block-style-select')).not.toBeVisible()
  })
```

Reply:
I have added the test @robinpyon 👍